README.md

```diff
@@ -8,3 +8,68 @@ Branches:
 * `XS-x.y` (e.g. `XS-7.5`), when they exist, contain sources from the `x.y` release of XenServer, with trademarked or copyrighted material stripped if needed.
 
 Built RPMs and source RPMs are available on https://updates.xcp-ng.org.
+
+---
+
+## :crystal_ball: Information & Remarks
+
+This Linux kernel module adds support to XCP-ng for NICs based on Marvell _(originally Aquantia)_ AQC111U over USB3. The driver should not be confused with the generic AQC111 that supports the whole family of NICs based on the AQC111 chipset, but NOT the ones connected over USB3. This kernel module provides support only for AQC111U-based NICs.
+
+The kernel module has been tested in a homelab environment with both XCP-ng version `8.1` and `8.2`.
+
+Known compatible NICs are<sup id="nt1">[1](#fnt1)</sup>:
+
+| Brand    | Model      | Tested             |
+|----------|:----------:|:------------------:|
+| QNAP     | QNA-UC5G1T | :white_check_mark: |
+| Sabrent  | NT-SS5G    | :white_check_mark: |
+| StarTech | US5GA30    |                    |
+| StarTech | US5GC30    |                    |
+| Trendnet | TUC-ET5G   |                    |
```

Comment:
The documentation, as you pointed out, is probably the right place, although until it's missing there it is an interesting chunk of information to have. How do you prefer me to address this?

- the changes should be removed and the `https://xcp-ng.org/docs/` site should have the modifications added? (How?)
- leave the info here and maintain it? _(it will not age well, but it will not age that fast I believe)_

Reply:
I definitely prefer to have it in the docs :)

Reply:
Commit 0f08735 removes this section in favor of a the [dedicated PR #97 in xcp-ng/xcp-ng-org documetation](https://github.com/xcp-ng/xcp-ng-org/pull/97)